testdata/templates/foo.yaml

```diff
@@ -6,6 +6,7 @@ routingKey: rk
 routingDelegate: rd
 thrift: foo.thrift
 timeout: 4.5s
+disableThriftEnvelope: false
```

Comment:
I wanted to make sure it was overriding any even when it's false, but decided to just make this `true` and `TestDisableThriftEnvelopeOverride` for testing the overrides.